modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -2196,9 +2202,6 @@ private <T> CompletableFuture<T> appendTxCommand(
                                         }))
                 );
             }
-        } catch (Exception e) {
```

Comment:
What will happen if cursor related exception will be thrown? Or it's not possible?

Reply:
It will just fly further. The code I removed would just add a wrapping exception around it, but the exception text could easily be misleading (as an exception can happen not just on an attempt to close the cursor)

Reply:
Makes sense.